activerecord/lib/active_record/enum.rb

```diff
@@ -195,8 +196,10 @@ def enum(definitions)
             define_method("#{value_method_name}!") { update!(attr => value) }
 
             # scope :active, -> { where(status: 0) }
-            klass.send(:detect_enum_conflict!, name, value_method_name, true)
-            klass.scope value_method_name, -> { where(attr => value) }
+            if enum_scopes.nil? || enum_scopes
```

Comment:
Better to change this to `if enum_scopes != false`

Reply:
ok.. will do